EthOn.rdf

```diff
@@ -1821,8 +1821,9 @@
     <owl:Class rdf:about="http://ethon.consensys.net/ContractAccount">
         <rdfs:subClassOf rdf:resource="http://ethon.consensys.net/Account"/>
         <owl:disjointWith rdf:resource="http://ethon.consensys.net/ExternalAccount"/>
+        <simpleDefinition>A type of Ethereum account that represents a smart contract. Its behaviour is controlled by the logik in the smart contract's code. Like external accounts it is identified by an address.</simpleDefinition>
         <suggestedStringRepresentation xml:lang="en">ContractAccount</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">A notional object existent only within the hypothetical state of Ethereum. Has an intrinsic address and thus an associated Account; the Account will have non-empty associated EVM Code. Incorporated only as the Storage State of that Account.</rdfs:comment>
+        <rdfs:comment xml:lang="en">One of the two main types of accounts on Ethereum, a contract account is an account whose behaviour is controlled by a smart contract. Contract accounts are identified by their address which is derived from the creator account's address and nonce. A contract account has a non-empty associated EVM code. It's state data consists of the bytecode, the contract's balance and the storage state of the contract's code (for example the value of variables). A contract account can only run and act when it is triggered by a message. It may not create and sign transactions, but it can receive transactions from external accounts as well as send and receive contract messages, which may involve a transfer of Ether. Contract accounts can also contain events which create log entries when triggered.</rdfs:comment>
```

Comment:
Excellent, this is much clearer. I just removed the words "run and" from the sentence which now reads "A contract account can only act when it is triggered by a message" as it seemed redundant.